hs_core/models.py

```diff
@@ -610,10 +610,13 @@ def update(cls, element_id, **kwargs):
                 except TypeError:
                     raise TypeError("Not a valid date value.")
                 if dt.type == 'created':
-                    raise ValidationError("Resource creation date can't be changed")
+                    #raise ValidationError("Resource creation date can't be changed")
+                    dt.start_date = start_dt
+                    dt.save(force_update=True)
                 elif dt.type == 'modified':
-                    dt.start_date = now().isoformat()
-                    dt.save()
+                    # dt.start_date = now().isoformat()
+                    dt.start_date = start_dt
+                    dt.save(force_update=True)
```

Comment:
> On Sep 18, 2015, at 1:05 PM, Hong Yi <notifications@github.com> wrote:
> 
> In hs_core/models.py <https://github.com/hydroshare/hydroshare/pull/598#discussion_r39877458>:
> 
> >                  elif dt.type == 'modified':
> > -                    dt.start_date = now().isoformat()
> > -                    dt.save()
> > +                    # dt.start_date = now().isoformat()
> > +                    dt.start_date = start_dt
> > +                    dt.save(force_update=True)
> @selimnairb <https://github.com/selimnairb> In resource landing page, "Created:" field should show the start_date you parsed out of the bag, right?
> 

Correct.

> The "Last updated:" should show the current date/time when the bag is ingested, right?
> 
No.  The last updated field should show the modification datetime stored in the bag; the bag ingestion should be transparent to users.
> Do you see "Created:" field not updated with the start_date you parsed out of the bag even though it shows updated value in your debugger?
> 

The created field is now correctly showing the original creation date of the bag.  It is the last updated that is not being updated in the web interface.